zyte_api/aio/client.py

```diff
@@ -65,6 +66,12 @@ async def request_raw(self, query: dict, *,
         auth = aiohttp.BasicAuth(self.api_key)
         headers = {'User-Agent': user_agent(aiohttp)}
 
+        # NOTE: Remove this check if the following commit for aiohttp which
+        # adds direct client support for brotli has been released after 3.8.1:
+        # https://github.com/aio-libs/aiohttp/commit/28ea32d2282728a94af73c87efd6ab314c14320e
+        if HAS_BROTLI:
+            headers['Accept-Encoding'] = 'br'
```

Comment:
I think we should be doing this even after that commit is released, i.e. we should always set `Accept-Encoding` to `br`, and not to `gzip, deflate, br`.

Also, since we are requiring `brotli` as a dependency, using `HAS_BROTLI` should not be needed.

Reply:
You're right. 👍 Updated!

Reply:
I think previously it was fine as well - e.g. in an unlikely event that aiohttp's HAS_BROTLI logic would become different from "brotli library is installed", the code would still have worked, although with gzip instead of brotli.

Reply:
Feel free to merge it though, it's not a big deal.